docs/ord.md

```diff
@@ -13,7 +13,18 @@ Example:
   "ord": {
     "namespace": "sap.sample",
     "description": "this is my custom description",
-    "policyLevel": "sap:core:v1"
+    "policyLevel": "sap:core:v1",
+    "products": [
+        {
+            "ordId": "sap:product:eb.bm.tests:",
+            "vendor": "sap:vendor:SAP:"
+        }
+    ],
+    "packages": [
+        {
+            "vendor": "sap:vendor:SAP:"
```

Comment:
please add a statement, that the overwrites have an effect on products not being generated and packages to reference this product.